### PR TITLE
updated readme and contributing guidelines

### PR DESCRIPTION
**Pull request by vlada-shubina**
### Solution
updated readme and contributing guidelines

- contributing guidelines to be in the repo root only. The guideline from the Wiki will be removed. Let's keep Wiki template authoring centric.
- added guidelines how to report the issues
- added guideline how to contribute
- updated readme.md, removed general help for `dotnet new.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
+| `Microsoft.TemplateEngine.Authoring.CLI` | Authoring tools: dotnet CLI tool with utilities for template authoring |
+| `Microsoft.TemplateEngine.Authoring.TemplateVerifier` | Authoring tools: [snapshot testing framework](docs/authoring-tools/Templates-Testing-Tooling.md) for the templates |
+
+### `dotnet new`
+
+`dotnet new` CLI is now located in [dotnet/sdk](https://github.com/dotnet/sdk/tree/main/src/Cli/Microsoft.TemplateEngine.Cli) repo.
+
+The issues for `dotnet new` CLI UX should be opened in the that repository.
 
+### Template Content Repositories
```
Comment (baronfel) [inline README.md:34]: I wish we could extract this out and reuse it in both documents

**Review by baronfel — APPROVED** [review]
This is great! Leaning into the GitHub experiences for contributing/readmes is always a good thing, and I appreciate the detailed scenario-based workflows you added.

**Review by JanKrivanek — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (vlada-shubina) [reply README.md:34]: https://github.com/github/markup/issues/346 +100500 + 1 user would agree with you :)

**Pull request merged**